src/pages/HorseRace/components/Back/Back_View.tsx

```diff
@@ -75,10 +77,22 @@ const BackView: React.FC<Props> = ({
               className="w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-300 focus:ring focus:ring-indigo-200 focus:ring-opacity-50 mb-2"
             />
           </label>
-
-          <span className="py-1">
-            Payout: <span>{potentialPayout}</span>
-          </span>
+          <div className="grid grid-cols-2 gap-4">
+            <div>
```

Comment:
no need for empty grid tag - u can add `block` to the `<span>` classname if you want it to behave like a div